init/start.sh

```diff
@@ -83,4 +83,5 @@ exec sudo -u ${CE_USER} -H --preserve-env=NODE_ENV -- \
     --env amazon \
     --port 10240 \
     --static out/dist \
+    --dist \
```

Comment:
So we'll need a new base image to get this working :| That's a bummer, but we'll live with it :) Worth it for this change.